client/init.js

```diff
@@ -4,9 +4,9 @@ addEventListener('load', function() {
 	var friendsList = null
 	var search = null
 	var destinationView = null
-
+	var timeoutId = null
+	var timeout = 30000 //30s
```

Comment:
I think we should tweek this a bit  to make it a bit more user friendly. 30 seconds works well, but I don't know if that is too much/little time.